integration-test-groups/cxf-soap/cxf-soap-client/src/main/java/org/apache/camel/quarkus/component/cxf/soap/client/it/CxfSoapClientRoutes.java

```diff
@@ -38,16 +47,45 @@ public class CxfSoapClientRoutes extends RouteBuilder {
     @ConfigProperty(name = "camel-quarkus.it.calculator.baseUri")
     String serviceBaseUri;
 
+    public static final String MESSAGE_RAW_SIMPLE_ADD = "<soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\" xmlns:ser=\"http://www.jboss.org/eap/quickstarts/wscalculator/Calculator\">\n"
+            +
+            "   <soapenv:Header/>\n" +
+            "   <soapenv:Body>\n" +
+            "      <ser:add>\n" +
+            "         <arg0>%s</arg0>\n" +
+            "         <arg1>%s</arg1>\n" +
+            "      </ser:add>\n" +
+            "   </soapenv:Body>\n" +
+            "</soapenv:Envelope>";
+
     @Override
     public void configure() {
 
         from("direct:simpleUriBean")
-                .to("cxf:bean:soapClientEndpoint?dataFormat=PAYLOAD");
+                .to("cxf:bean:soapClientEndpoint?dataFormat=POJO");
```

Comment:
Why this change? I thought we are only adding tests here and not changing any thing that worked before. Or this did not work before?

Reply:
It was working only by accident. With change to @SessionScoped of the `soapClientEndpoint` it has to be `POJO` mode. `PAYLOAD` mode was never used in that case.

Reply:
More info in this https://github.com/apache/camel-quarkus/issues/4282#issuecomment-1321228549